### DISCUSSION TITLE
Aleph presentation / non-restricted data mode

### DISCUSSION THREAD

**Discussion by kjacks** [alephdata/aleph#1875] [Ideas]
Allow a user to toggle between viewing all data they have access to vs. only data from non-restricted datasets (Presentation mode).

Since we already have a restricted flag on the datasets, it seems like it could be a fairly light lift to support this as a filter in the request.

We could add some visual cues to the UI to remind someone that they are browsing in presentation mode, like a border / similar to how Google Meet reminds you that you are sharing your screen.

It could be hidden behind the tester flag to potentially avoid confusion for new/ less familiar users.

**Comment by Rosencrantz** [comment]
How hard do you think this would be to implement?

**Reply by kjacks** [reply]
The ui side of things seems fairly minimal (a settings toggle and some kind of border to remind you you're in restricted mode). In my mind the complexity of this lies more on the backend? @sunu
